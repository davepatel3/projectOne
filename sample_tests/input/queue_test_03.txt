int
5
4
2
eb -41
gs
eb -96
df
ef -70
ef 67
pb
df
ef 8
eb 46
df
pf
ef 27
eb -42
cl
gc
ps
ef -2
eb 1
eb 98
gb
ef 14
df
gs
ps
db
gs
gc
ef -39
ef -50
ef 51
gc
pb
gs
pb
cl
gc
gf
db
ps
eb 72
gf
gs
ps
eb 29
eb 42
ps
ps
ps
cl
db
gs
eb 49
df
pb
db
df
gs
ef 16
eb -18
eb 97
ef -33
df
gb
ps
ef 28
ef 82
db
qt
