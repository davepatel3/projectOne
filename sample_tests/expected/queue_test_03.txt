1
-41
67
8
5
[-][-][-][-][-]
98
14
3
[+][+][-][-][+]
98
2
5
5
4
5
cyclic_double_queue is empty!
cyclic_double_queue is empty!
[-][-][-][-][-]
72
1
[+][-][-][-][-]
[+][+][+][-][-]
[+][+][+][-][-]
[+][+][+][-][-]
cyclic_double_queue is empty!
0
49
cyclic_double_queue is empty!
cyclic_double_queue is empty!
cyclic_double_queue is empty!
0
-33
97
[+][+][+][-][-]
97
